### ISSUE TITLE
TKLBAM: support for vanilla Debian OS

### ISSUE THREAD

**Issue by JedMeister** [turnkeylinux/tracker#1128]
A non-TurnKey TKLBAM user reported that he had issues with TKLBAM on (vanilla) Debian Stretch:

> A few issues with https://github.com/turnkeylinux/tklbam on stretch
>
> ez-apt-install.sh
>
> Doesn't list 
>
> 9.*) echo stretch ;;
>
>Fails on apt-key dependency on dirmngr (clean base install)
>
> Fails on pgpkeys.mit.edu down for maintenance, works on...
>
> apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys A16EB94D
>
> TurnKey Hub Error: Backup profile archive not found: debian-9

**Comment by Dude4Linux** [comment]
@JedMeister - I'll add what I've experienced here rather than opening a new issue because I believe there is a common cause.  After installing the most recent updates, I started having problems with my v15 tkldev appliance.  Thinking they might be caused by the way I created the appliance starting with v14.2 and upgrading, I decided to start fresh with a new install of turnkey-tkldev-15.0 downloaded as an iso and installed in a ProxMox VM.

One of the custom appliances I was trying to build uses a third-party repo for which I need to install an apt-key.  The errors reported by make were:
```+ apt-key adv --keyserver keyserver.ubuntu.com --recv 06E85760C0A52C50
Executing: /tmp/apt-key-gpghome.OlMmbdgVoA/gpg.1.sh --keyserver keyserver.ubuntu.com --recv 06E85760C0A52C50
gpg: failed to start the dirmngr '/usr/bin/dirmngr': No such file or directory
gpg: connecting dirmngr at '/tmp/apt-key-gpghome.OlMmbdgVoA/S.dirmngr' failed: No such file or directory
gpg: keyserver receive failed: No dirmngr
/usr/share/fab/product.mk:535: recipe for target 'build/stamps/root.patched' failed
make: *** [build/stamps/root.patched] Error 2
```
Googling for an answer gave https://blog.sleeplessbeastie.eu/2017/11/02/how-to-fix-missing-dirmngr/.
Apparently in Debian stretch the `gnupg` package changed `dirmngr` from required to recommended because some use cases for `gnupg` don't require network access.  Since most TurnKey applications do need network access, I think it would be a good idea to add `dirmngr` to the list of core packages.
